spec/support/factory_girl.rb

```diff
@@ -2,6 +2,7 @@
 
 FactoryGirl.define do
   factory :user do
+    name 'Charlotte'
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.